web-animations-1/Overview.bs

```diff
@@ -3201,6 +3201,11 @@ If the [=effect target=] is a [=pseudo-element=], the [=target element=] is
 its [=originating element=] and the [=target pseudo-selector=] is as required
 to specify that particular [=pseudo-element=].
 
+Note that not all [=effect targets=] specified in this manner (such as ''::part()''
```

Comment:
I think this is attempting to address https://github.com/w3c/csswg-drafts/issues/4701, right?

I'm not opposed to this approach, though I wonder if we should just have an early exit in `5.3.3. The effect value of a keyframe effect` instead.

Reply:
I kept this and added reminders to the other procedures just like `5.3.2. Calculating computed keyframes` has.

Reply:
moot